src/utils/runtime/src/variables.rs

```diff
@@ -0,0 +1,27 @@
+// Copyright 2023 RisingWave Labs
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use std::sync::OnceLock;
+
+static SERVER_START_TIME: OnceLock<chrono::DateTime<chrono::Utc>> = OnceLock::new();
+
+/// Init the server start time, should be called only once.
+pub(crate) fn init_server_start_time() {
+    SERVER_START_TIME.get_or_init(chrono::Utc::now);
+}
```

Comment:
What about directly use `LazyLock::force` here? In which case we'll expect `SERVER_START_TIME` to be none?

Reply:
Depends on the entrypoint. I'm unsure about how it'll be called.